dist/riot-gitconfig

```diff
@@ -0,0 +1,4 @@
+[user]
```

Comment:
Is this file needed ?

Reply:
Which file ? I do not see any file there :p